site/gdocs/centered-article.scss

```diff
@@ -70,6 +70,19 @@ $banner-height: 200px;
     margin-top: 0;
 }
 
+.article-block__blockquote {
+    margin-top: 0;
+    p {
+        font-style: italic;
+        &:last-of-type {
+            margin-bottom: 4px;
```

Comment:
If we only need this margin if the footer/cite block is present, it might be better to make it the footer's responsibility to apply this margin?